index.html

```diff
@@ -1,12 +1,147 @@
     
+    <div class="mobile-menu" id="mobileMenu">
+        <img id="menu-close" src="images/closeIcon.png" alt="close icon">
+        <ul class="mob-menu">
+          <li class="menu"><a class="moblink-menu" href="./index.html">Home</a></li>
+          <li class="menu"><a class="moblink-menu" href="./about.html">About</a></li>
+          <li class="menu"><a class="moblink-menu" href="#about.html">Program</a></li>
+          <li class="menu"><a class="moblink-menu" href="#contact-me">Join</a></li>
+          <li class="menu"><a class="moblink-menu" href="#about-me">Sponser</a></li>
+          <li class="menu"><a class="moblink-menu" href="#contact-me">Register</a></li>
+        </ul>
+    </div>
+    
+     </div>
+  </nav>
+    <div class="front">
+        <div class="front1">
+        <h2>"hello! developers"</h2>
+        <h1>SOFTWARE DEVELOPERS <br> GLOBAL SUMMIT 2023</h1>
+        </div>
+        <div class="front2">
+        <p class="pbox1">A motivational Conference for those who want to be profecional software developer held by Technology for Africa is taking place in June Ethiopia.</p>
+        </div>
+        <div class="front3">
+        <h3 class="date">2023.10.15(THO) ~ 16(FRI)</h3>
+        <p>@AU Assemble Hall, ADDIS ABABA ETHIOPIA</p>
+        </div>
+    </div>
+</section>
+</header>
+    <section class="mainpage">
+        <h1>Main Page</h1>
+         <hr class="redline">
+        <div class="boxes">
+        <div class="box">
+            <div class="photo"><img src="images/images (4).png" alt=""></div>
+           <h3>Lecture</h3>
+           <p>Listen from the speakers those who came from all over the world and who are profeciaonal in software development </p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (1).png" alt=""></div>
+            <h3>CC Exhibition </h3>
+            <p>Appreciate various creations applying software development licens of ariists organized from Art Center Nabi</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (2).png" alt=""></div>
+            <h3>Forum</h3>
+            <p>Have the same time to share to your thoughts, opinions with experts of software developers topic.</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images (3).png" alt=""></div>
+            <h3>Workshop</h3> 
+            <p>Try Creating your own apps using open source license rather than just watching about software app.</p>
+        </div>
+        <div class="box">
+            <div class="photo"><img src="images/images.jpg" alt=""></div>
+            <h3>CC Ignite</h3>
+            <p>Get the  chance to network with CC affilliates around the world ,also after the summit of development</p>
+        </div>
+     </div>
+     <div class="redbox"> <h3> <a href="#">View full Program></a> </h3> </div>
+    </section>
+    <div class="feature">
+        <h1>Main Speakers</h1>
```

Comment:
**OPTIONAL** - As i suggested above, please avoid having more than one `h1` tag in your HTML page